src/SortingAlgorithms/SORTING.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Do not add empty markdown files, Make a list of href links to all the sorting algos you are specifying.